rdm/0006-profiles-and-flavours.md

```diff
@@ -0,0 +1,104 @@
+- Start Date: 2019-10-02
+- RFC PR: [#6](https://github.com/inveniosoftware/rfcs/pull/6)
+- Authors: Guillaume Viger
+
+# Customize created Invenio instance by choosing profile and optionally flavour
+
+## Summary
+
+When running the instance bootstrapping CLI command (`invenio-cli init`), the
+hoster should be given a choice to create an Integrated Library System (ILS) or
+Resource Data Management (RDM) "flavour" Invenio application. If RDM is chosen,
+there are further "profiles" to be chosen from, for instance "default" and
```

Comment:
We sometimes talk about default, generic, institutional repository. I think this is a terminology that might be defined. Shall it go to another RFC?

Reply:
I've added a quick summary of the profiles in the terminology to address this.